Sources/PostgreSQL/Connection/PostgreSQLConnection+Connect.swift

```diff
@@ -0,0 +1,57 @@
+import Async
+import NIO
+import NIOOpenSSL
+
+extension PostgreSQLConnection {
+	/// Connects to a PostgreSQL server using a TCP socket.
+    public static func connect(
+		to serverAddress: PostgreSQLDatabaseConfig.ServerAddress = .default,
```

Comment:
There appears to be a formatting issue here.

Reply:
Done, thanks! Fixed whitespace everywhere.